examples/bankaccount/bank_wadm.yaml

```
@@ -0,0 +1,74 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: bank-account
+  annotations:
+    version: v0.0.1
+    description: "A concordance bank account example"
+spec:
+  components:
+    - name: projector
+      type: actor
+      properties:
+        image: file://./projector/build/bankaccount_projector_s.wasm
+      traits:
+        - type: spreadscaler
+          properties:
+            replicas: 1            
+        - type: linkdef
+          properties:
+            target: concordance
+            values:
+              ROLE: projector
+              INTEREST: account_created,funds_deposited,funds_released,funds_reserved,funds_withdrawn,wire_transfer_initiated
+        - type: linkdef
+          properties:
+            target: redis
+            values:
+              URL: redis://redis:6379
```

```suggestion
              URL: redis://127.0.0.1:6379
```

Would suggest we point at localhost instead of assuming Docker, unless there's a docker-compose that I'm blind to